src/lib/types.ts

```diff
@@ -61,3 +63,21 @@ export enum PublicEvents {
 export enum InternalEvents {
```

Comment:
Move that out of this file so that it's not part of the official public exposed API

Reply:
Fixed